### ISSUE TITLE
Error after update to FW 2.0.0 (not beta) 

### ISSUE THREAD

**Issue by isynx** [danielkraak/GC-Loader#32]
Today I updated the FW from 2.0.0 beta to 2.0.0 but then got error after rebooting saying:
An error has occured.

Turn the power OFF and check the NINTENDO GAMECUBE Instruction Booklet for further instructions.

**Comment by dejones972** [comment]
i have the same problem.

**Comment by isynx** [comment]
I wonder is there any way to downgrade easily as I already removed the optical drive and my gamecube has no serial 2 port...

**Comment by tjanas** [comment]
@danielkraak is there a way to force gc-loader to downgrade to a different firmware version upon system boot?

**Comment by cvgc9** [comment]
same problem .

**Comment by matt456** [comment]
same issue here, latest Swiss prompted a GC Loader firmware update was available. Firmware update went as expected, it prompted to turn on/off Gamecube. Now it won't boot swiss or any other iso set to boot.iso on the SD card. Tried an SD card from another working GC Loader that still failed.

**Comment by tjanas** [comment]
Does anyone know how to get a hold of the developer of GC Loader?

**Comment by danielkraak** [comment]
Unfortunately, I do not believe it is possible to downgrade the firmware through the GameCube. I have received back one board  that also no longer worked after the update, so I will look into this issue.

If you bought a GC Loader from gc-loader.com, feel free to use the contact form on the website for help with resolving the issue.

**Comment by danielkraak** [comment]
Also, sorry for the unresponsiveness. Usually I also receive a message through the contact form on the website to resolve these issues.

**Comment by tjanas** [comment]
Has anyone here had the issue with one ordered from Black Dog Technologies? For those with the issue, where did you obtain it?

**Comment by cvgc9** [comment]
I ordered mine from Black Dog Technology. Worked fine before update 2.0.0. Tried multiple sd cards and installed in different GameCube . Wouldn’t boot .

**Comment by AlexUnderBase** [comment]
I have tried Mario Kart after upadate from 1.1.2 to 2.0.0 and all went good ! My unit is from 1st batch sold on Black Dog Technologies , the one who ive did implant from cd drive of the connector to GcLoader unit 

![Fdt8WkNX0AEkXFE](https://user-images.githubusercontent.com/38410435/197149412-076b2491-61de-43a6-99f3-b59d5dfeca8f.jpeg)

**Comment by isynx** [comment]
Maybe updating from 1.2 to 2.0 is okay but not from 2.0beta
Mine is also from black dog tech but I doubt if they will provide any easy solution

**Comment by tjanas** [comment]
@isynx have you contacted Dan at Black Dog Technologies to let him know about this issue?

**Comment by isynx** [comment]
I just wrote an message to them finger crossed

**Comment by isynx** [comment]
While still waiting for balck dog tech to reply, I wonder if by installing a picoboot can solve the problem. If I mod the console via picoboot, then I maybe able to boot and update the gcloader? Anyone has tried this and cares to share?

**Comment by danielkraak** [comment]
This most likely will not work. I have one GC Loader in my possession that does not work with 2.0.0 and it seems to lock out. As a result, it will not be able to process the update commands. 

I will be looking into why it locks out in order to fix the issue.

**Comment by Hagen76** [comment]
I'll stay in 2.0.0 beta on my two gc loaders until it will be safe to update^^ I really don't want to lock it.
I've bought them from gc-loader.com!

**Comment by tjanas** [comment]
Any updates? Any work from Black Dog Tech?

**Comment by isynx** [comment]
> Any updates? Any word from Black Dog Tech?

No reply so far. I would never update mine if given a second chance:s

**Comment by Gitomata** [comment]
> Any updates? Any word from Black Dog Tech?

I think his pet dog passed away recently so he might be occupied atm. I think it's the same dog that appears on shop's logo.

**Comment by Hagen76** [comment]
Any updates?

**Comment by isynx** [comment]
No replies so far. I am going to do a piboot instead

**Comment by elgarlic** [comment]
I had this problem with a previous GCLoader firmware update. "Turn the power OFF and check the NINTENDO GAMECUBE Instruction Booklet for further instructions." Dozens of attempted boots failed, while it had worked great on the previous firmware. I was told, "I'm guessing the new firmware doesn't meet timing constraints on your FPGA." (silicon lottery).

I unplugged the console for several hours and managed to get it to boot a single time. After that, it was back to failed boots. Repeated the process, unplugging, this time with a firmware downgrade in place on the memory card. Booted, downgraded, and working again. It may have been coincidence, as I was also told that unplugging should have had no effect. But it seems statistically unlikely that the two successful out of many dozens of attempts were done that way. 

The next update was 2.0.0b, which was fine and is still working for me. Staying the hell away from 2.0.0 final. :)

**Comment by j117-esc** [comment]
I was about to upgrade until I saw this post. I guess we better stay on 2.0.0.BETA  from the look of things.

**Comment by g5star** [comment]
I have had the same problem and I got mine from AliExpress. I believe I tried 2.0 version with mine and got the same results. I got error messages and now i get a dark blue screen on the tv. The GC Loader is the lite version It is V0.7 and was made in 2020. Does anyone know if you can reflash the Gc Loader?

**Comment by elgarlic** [comment]
> I have had the same problem and I got mine from AliExpress. I believe I tried 2.0 version with mine and got the same results. I got error messages and now i get a dark blue screen on the tv. The GC Loader is the lite version It is V0.7 and was made in 2020. Does anyone know if you can reflash the Gc Loader?

The device you're talking about is a failed clone. They named it a GCloader lite because they couldn't even get it working to load games from its SD card, but they wanted to sell it anyway. So it should not be surprising if it fails to update or bricks, and it's not right to request support for clones from the developer.

**Comment by tqhoang84** [comment]
I have an new-in-box GCLoader PNP (from BDT).  Can the existing firmware be backed up & restored via a CH341A programmer?  Any jumper to short/bridge first (like JP1 on the GCHD MK2)?

**Comment by tjanas** [comment]
Bump

**Comment by tqhoang84** [comment]
> I have an new-in-box GCLoader PNP (from BDT). Can the existing firmware be backed up & restored via a CH341A programmer? Any jumper to short/bridge first (like JP1 on the GCHD MK2)?

@danielkraak - Is the firmware able to be backed-up & restored via a CH341A programmer?

**Comment by abeisgoat** [comment]
Just another data point - my GC Loader was bricked from doing the 2.0.0 update as well, tried many restarts with no luck. I've sent an email so we'll see how it is handled.

**Comment by snowsensei** [comment]
I went from 1.1.2 to 2.0 and, while it didn't brick my GCLoader, it created an issue where it would only boot into Swiss about one in 5 times I turned the console off and on again. I finally downgraded to the 2.0 beta and things have been fine. If it wasn't for the annoying notification that tells me there's an update available every time I turn it on, I'd probably never touch the firmware again.

**Comment by elgarlic** [comment]
> I went from 1.1.2 to 2.0 and, while it didn't brick my GCLoader, it created an issue where it would only boot into Swiss about one in 5 times I turned the console off and on again. I finally downgraded to the 2.0 beta and things have been fine. If it wasn't for the annoying notification that tells me there's an update available every time I turn it on, I'd probably never touch the firmware again.

Yeah, the obnoxious nag screen is often the main downside. It should either appear only once or have an option to disable.

Apparently you can normally prevent it by editing the .ini. But of course the one exception to this seems to be 2.0.0.beta. https://www.reddit.com/r/Gamecube/comments/z4el6w/swiss_can_you_hide_the_gc_loader_firmware_update/

**Comment by abeisgoat** [comment]
Follow up from my last comment for anyone curious, after my GC loader was bricked by this update I went ahead and bought the $15 in parts to Picoboot it and that worked flawlessly and confirms that the GC Loader does not show up as a disk drive at all any more. Attempting to boot the GC Loader upgrade (or downgrade as it were) just freezes out.

**Comment by tjanas** [comment]
Has Black Dog Tech responded to this issue yet?

**Comment by abeisgoat** [comment]
> Has Black Dog Tech responded to this issue yet?

@danielkraak last responded in October and has not replied to my email.

**Comment by DIYdotexe** [comment]
my GC loader was just bricked today by the 2.0.0 update. Why was it still available for download after it was a known issue?
Has anybody heard from kraak or black dog?

**Comment by elgarlic** [comment]
> my GC loader was just bricked today by the 2.0.0 update. Why was it still available for download after it was a known issue? Has anybody heard from kraak or black dog?

Sorry to hear that. No response. They've got none to sell due to the parts shortage. So it's not even a high enough priority to take the link down or offer a warning.

**Comment by DIYdotexe** [comment]
Uhh, if that’s true then it says a lot about how much they care about their customers

**Comment by isynx** [comment]
I guess this is a pitfall of all DIY projects. The maintenance cost is too high or the work is tedious. I still do hope there would be a solution but up to now it seems not very likely. Putting a warning besides the firmware 2.0 is definitely helpful to others who blindly update their firmware like me:p

**Comment by JoeTheRetro** [comment]
I was running the official 2.0.0 just fine, updated a couple of weeks ago for no reason other than noticing there was a new firmware. Dan posted on Twitter today and someone mentioned this problem. I just finished downgrading back to 1.1.2. I never had any problem on that firmware, so I figure better safe than sorry. What a bummer for those who have been bricked. I have a day one launch GC Loader. This must be affecting certain batches.

**Comment by citrus3000psi** [comment]
This seems to isolated problem which is why the firmware is still up.  It could also be related to the update process itself.  If you have purchased from black-dog and have a bricked 2.0 unit.  Send me an email to me@dankunz.com and I will get you squared away.

**Comment by tjanas** [comment]
@isynx can you re-open this issue, since upgrading to 2.0.0 still has issues and there isn’t a newer firmware yet from @danielkraak to fix those problems?

**Comment by snowsensei** [comment]
Agreed. Especially since, for people who didn't buy direct from Black Dog, there is no fix.

**Comment by isynx** [comment]
I would like to reopen the issue but I don't find a way to do it. Maybe it is better to open a new issue. In my case, Dan sent me a replacement (I believe so) which has been updated to the latest firmware. Thus, my guess is that this is more hardware related lockup than something which can be fixed through an update. So try contact Dan and be patient, he will sort you out as long as you have a genuine gcloader.

**Comment by danielkraak** [comment]
I am looking into solving this issue. I will also be adding additional recovery functionality, so it should always be possible to recover from this issue in the future.

**Comment by KirovAir** [comment]
I found this comment on reddit, it seems you can still downgrade your GCLoader even when it's bricked:
`Hey! I actually had the same issue and got the system to reflash the GCLoader. I put the disc drive back in and booted into Swiss via Paper Mario and a Hacked Save. Once I got into Swiss, I opened the PREVIOUS (not beta) update file with the disc drive still connected. The update software will detect firmware on the disc drive and then be ready to update At this point, I hot swapped the disc drive for the GCLoader. Then once it was connected, I pressed A, and it reflashed the GCLoader board with the previous firmware. Just be careful not to flash firmware while your disc drive is attached. Good luck!`

**Comment by caprisun08** [comment]
> 

This is great! However I still would have to set up the exploit save with a modded wii correct? Is it possible to mount a pre existing exploited memory card that someone else has prepped? I do not have a modded wii close by.

**Comment by danielkraak** [comment]
I have just uploaded version 2.0.1.BETA to the repository. I am pretty positive that this will fix the issues. Black Dog Tech and I have tested this firmware on returned boards and all of them worked with this firmware. 

It also has additional recovery options that can also recover from random crashes (as was happening 2.0.0, but they should also be gone now with this firmware). 

I will keep the firmware BETA for now, but can hopefully kick it out of BETA soon. If there still happen to be problems, do not hesitate to contact me (gc-loader.com) or black dog tech and we will get your unit fixed.

**Comment by snowsensei** [comment]
Working great for me, thanks!

**Comment by tjanas** [comment]
> I will keep the firmware BETA for now, but can hopefully kick it out of BETA soon. If there still happen to be problems, do not hesitate to contact me (gc-loader.com) or black dog tech and we will get your unit fixed.

Can you make a notification that it is now out of BETA? https://github.com/danielkraak/GC-Loader/issues/33

**Comment by Shindur** [comment]
Just wanted to comment that 2.0.1 release worked flawlessly on my GC Loader. It was having issues with 2.0.0, luckily I was able to revert it back to 1.1.2 until this new release came. Thank you everyone who worked on it!

**Comment by pkbroot** [comment]
Just got and installed my GC Loader HW2 today and I can't update. It seems to think that firmware "2.0.0.GCLOADER_HW2" is higher than "2.0.1.BETA" perhaps? When I launch the update it just shows the two top lines that display the update and the current detected firmware, but does not give the prompt of press A to start update process. However I still have a notification that a new firmware is present every time I boot up and launch into Swiss. Any suggestions?

**Comment by kurohouou** [comment]
I just got my two GC Loaders.. I have tried both of them and have the exact same issues on both..

1.  Can't upgrade the firmware from 2.0.0 to 2.0.1, same issue as pkbroot apparently.
2.  I notice as the system boots I see a quick message on the screen, looks like an error message in a red box (like the box it shows your resolution), it says 0x 0i60.. Not sure what that is, but it only flashes for a second then goes to Swiss.
3. I can't get it to boot my Sandisk 1TB SD Card  (https://a.co/d/cdFC5bb). Works with my 512GB Samsung Evo though.  Used Fat32 (even tried exFAT).
4. Last, the reboot button isn't work, I can be in a game and hit reboot and it just goes to a black screen and nothing else happens.

I emailed Dan as well, hopefully he will get this sorted quickly :)  Looking forward to playing some games!

**Comment by tjanas** [comment]
@pkbroot @kurohouou it sounds like you should report this in a different issue, since this issue here was for upgrading the original version of GCLoader. Your issues seem to be related to HW2 being incompatible with firmwares made for the original GCLoader.

**Comment by pkbroot** [comment]
Good idea, I did that.

**Comment by kurohouou** [comment]
Yeah, Dan said 2.0.1 isn't for the new hardware so that solved that problem.. Only issues I have really now are,

1.  My 1TB Sandisk MicroSD card can't be read, waiting on a new one from Amazon to test that, but I heard the older GCLoaders work with 1TB so hoping its just too new of a card maybe??
2.  The reboot button on the console itself doesn't work, I put my OG game controller ports back in (had the blueretro in before) but had the same issue so its not that.  That said, the software reboot works in game and honestly thats probably more important ;)
3. 
But ill stop commenting here as these issues arent related to this bug really.. so moving on.. just wanted to give a quick update incase someone comes across this.

**Comment by tjanas** [comment]
Today I directly updated from 1.1.2 to 2.0.1 (not BETA) without any issues (DOL-001 (USA), IPL 1.1).

**Comment by philivy** [comment]
Today I directly updated from 1.1.2 to 2.0.1 (not BETA) with issues (DOL-001 (EUR)). 
flashing  didn't succed and get at reboot the message 
"Turn the power OFF and check the NINTENDO GAMECUBE Instruction Booklet for further instructions.
so great upgrade !!! :D

**Comment by danielkraak** [comment]
Above person was using a bitfunx clone. We do not provide support for these.

**Comment by fwtooley7** [comment]
> I have just uploaded version 2.0.1.BETA to the repository. I am pretty positive that this will fix the issues. Black Dog Tech and I have tested this firmware on returned boards and all of them worked with this firmware.
> 
> It also has additional recovery options that can also recover from random crashes (as was happening 2.0.0, but they should also be gone now with this firmware).
> 
> I will keep the firmware BETA for now, but can hopefully kick it out of BETA soon. If there still happen to be problems, do not hesitate to contact me (gc-loader.com) or black dog tech and we will get your unit fixed.

Did not work for me! Still continue to have a bricked GC with no findable solution

**Comment by tjanas** [comment]
> 
> Did not work for me! Still continue to have a bricked GC with no findable solution

Where did you purchase your GCLoader from?
